src/stores/useLabyrinthStore.jsx

```diff
@@ -0,0 +1,89 @@
+import { create } from "zustand";
+
+export const useLabyrinthStore = create((set) => ({
+  userName: "",
+  loading: false,
+  error: null,
+  actions: [],
+  coordinates: null,
+  description: null,
+  gameMode: false,
```

Comment:
What does this mean? Could it be named hasStarted or isGameOn or something, if it has to do with that?